.github/workflows/lint.yml

```diff
@@ -188,14 +137,17 @@ jobs:
       - name: Checkout the Source Code
         uses: actions/checkout@v3
 
-      - name: Install Cargo-audit with Nightly Rust
+      - name: Set up Rust
         uses: ./.github/actions/setup-rust
         with:
           toolchain: nightly
           install: cargo-audit
-          cache: true
-          cache-job-id: ${{ github.workflow }}-${{ github.job }}
-          cache-hash: ${{ hashFiles('.github/workflows/core.yml') }}
+
+      - name: Install cargo-audit
+        uses: actions-rs/cargo@v1
+        with:
+          command: install
+          args: cargo-audit
```

Comment:
Isn' the previous command with `install: cargo-audit` doing that already?

Reply:
Oops